docs/source/user-guide/general/drift_guide.rst

```diff
@@ -0,0 +1,186 @@
+seasonal changes are usually not considered drift, as they can be predicted in advance and the model can learn from
+past data samples.
+
+
+Which Types of Drift Are There?
+================================
+
+In machine learning, we usually refer to 3 types of drift:
+
+Concept Drift
+-------------
+Concept drift happens when the underline relation between the data and the label has changed. In layman terms, this means that our
+data has changed in a way that affects our label, meaning a model trained on previous data will have difficulty predicting
+on the new data, as the label can't be predicted in the same way anymore.
+
+Concept drift will almost always require some changes to the model, and will require retraining it on newer data.
+
+Data Drift
+----------
+Data drift is any change in the distribution of the data. It usually refers to changes that do not significantly affect the label.
+However, even when not affecting the model, data drift is important, as it can expose other undetected issues.
+
+Moreover, when labels are not available (as happens in many cases), data drift cannot be discerned from concept drift.
+
+Label Drift
+-----------
+Label drift is the change in the distribution of the label itself. Note that this can also be caused by concept drift,
+but here we discuss a change that means that the label can still be accurately predicted from the data, but its
+distribution has changed.
+
+(Is this interesting? Should I add this to each drift? Should I detail the causes more?)
+
+Label drift is important for 2 reasons:
+
+1. Your model can be improved - Machine learning models use the label's distribution when learning to predict it.
+For example, if a dataset has 90% of label A and 10% of label B, the model may learn that label A is more common and
+is a better prediction in cases where the label can't be inferred easily. Changes to this distribution can prompt us
+to retrain the model on the new distribution, in order to more accurately simulate the current reality.
+Note that this is not relevant when the training dataset is sampled so labels are evenly distributed.
+
+2. You may need to take some action - Not all changes in data means we need to retrain our model.
+However, we may need to act differently on it.
+For example, a fraud prevention company predicts fraudulent payments in 80% accuracy. Suddenly, the company notices that
+that they are getting more fraudulent payments recently. They are still predicted in 80% accuracy, but those 20% they
+get wrong are now more costly, and therefore the company needs to adjust its actions - perhaps adjusting the model
+decision threshold, or adding a manual review process.
+
+For more on the different types of drift, `see here <https://deepchecks.com/data-drift-vs-concept-drift-what-are-the-main-differences/>`_
+
+
+What Are the Causes of Drift?
+==============================
+
+(Is this interesting on its own? Or should this be a part of the description of each drift?)
```

Comment:
Think that giving an example of change in the real world and change in data acquisition and processing in the first part is enough.